docs/src/test-fixtures.md

```diff
@@ -335,3 +335,41 @@ const test = base.extend<{}, ExpressWorkerFixtures>({
 
 export default test;
 ```
+
+## Augmenting fixtures
```

Comment:
I'm not a native english speaker, and think we should use probably a different word for that. I would propose one of the following ones:

- overwrite
- redefine

Maybe you have a better idea.

Reply:
Overwrite was one of the words I considered, the only part I didn't like was the fact that it gives the idea of replacing the existing fixture rather than layering logic on top of the existing fixture.  That might be overcomplicating it though and overwrite might be the best term 🙂

Reply:
Inheritance might be also a good candidate.

Reply:
How about extending fixtures ?

Reply:
I kind of like extending, but wonder if that is confusing given `test.extend` 😕 

Inheritance might work

Reply:
Let's use the "override" word, because you are essentially overriding the fixture with your own. Let's also mention (and maybe give an example):
- you can depend on other fixtures (good example would be `page` fixture that depends on `baseURL`);
- you can depend on the same fixture (as you do already in this example) to "extend" it;
- you can ignore the base fixture and return something entirely different (good example would be overriding the `storageState` fixture and provide your own data).

Reply:
Awesome, thanks for the info.  I'll go ahead and update this PR tomorrow to use override as well as add the other examples you mentioned.

Reply:
I updated the docs to what I think covers the points you made.  Please take a look and see if it makes sense.